examples/ex_bitmap_flip.c

```diff
@@ -138,6 +157,9 @@ int main(int argc, char **argv)
             if (event.keyboard.keycode == ALLEGRO_KEY_ESCAPE)
                done = true;
             else if (event.keyboard.keycode == ALLEGRO_KEY_SPACE) {
+				/* Spacebar toggles whether render from a memory bitmap 
+				 * or display bitamp.
+				 */
```

Comment:
You gotta watch your spaces and tabs here.  3 spaces instead of tabs.

I don't know what editor you're using; I use VIM and had to setup the configuration so that whenever I use `TAB` it actually replaces it with 3 spaces.